openlibrary/plugins/upstream/borrow.py

```diff
@@ -216,7 +216,7 @@ def POST(self, key):
 
     def POST_join_waitinglist(self, edition, user):
         waitinglist.join_waitinglist(user.key, edition.key)
-        raise web.redirect(edition.url("/borrow"))
+        raise web.redirect(edition.url())
```

Comment:
Is this right? Will the Waitlist Join still happen if it doesn't post to the `borrow.py` page?